### PR TITLE
Pw 666 checkout payment v40 and Application info 

### PR DESCRIPTION
**Pull request by AlexandrosMor**

### CODE REVIEW

File: .idea.bak/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rkewlani) [inline .idea.bak/compiler.xml]: Can you please remove all .idea/* files ?

Comment (AlexandrosMor) [reply .idea.bak/compiler.xml]: yes the files are removed

File: src/main/java/com/adyen/service/resource/payment/Authorise3DS2.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.adyen.service.resource.payment;
+
+import com.adyen.Client;
+import com.adyen.Service;
+import com.adyen.service.Resource;
+
+public class Authorise3DS2 extends Resource {
```
Comment (Aleffio) [inline src/main/java/com/adyen/service/resource/payment/Authorise3DS2.java]: is this used?

**Comment by rikterbeek** [comment]
@AlexandrosMor please remove all autogenerated code starting with @javax.annotation.Generated

File: src/main/java/com/adyen/model/SDKEphemPubKey.java [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+ */
+package com.adyen.model;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.Objects;
+
+/**
+ * SDKEphemPubKey
+ */
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.java.JavaClientCodegen", date = "2018-12-06T14:10:59.920Z[GMT]")
```
Comment (rkewlani) [inline src/main/java/com/adyen/model/SDKEphemPubKey.java]: Please remove swagger autogenerated annotations.

File: src/main/java/com/adyen/model/checkout/ModelConfiguration.java [hunk 4]
```diff
@@ -21,35 +21,84 @@
...
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.JsonAdapter;
 import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
 
-/**
- * ModelConfiguration
- */
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.java.JavaClientCodegen", date = "2018-12-11T13:51:52.335Z[GMT]")
```
Comment (rkewlani) [inline src/main/java/com/adyen/model/checkout/ModelConfiguration.java]: same as above, remove swagger annotation

File: src/main/java/com/adyen/model/checkout/PaymentMethodsRequest.java [hunk 5]
```diff
@@ -52,12 +57,99 @@
     private List<String> allowedPaymentMethods;
     @SerializedName("blockedPaymentMethods")
     private List<String> blockedPaymentMethods;
+    @SerializedName("additionalData")
+    private Object additionalData = null;
+
+    @SerializedName("accountInfo")
+    private AccountInfo accountInfo = null;
+
+    @SerializedName("applciationInfo")
```
Comment (rkewlani) [inline src/main/java/com/adyen/model/checkout/PaymentMethodsRequest.java]: ApplicationInfo is not applicable for PaymentMethodRequest

File: src/main/java/com/adyen/model/checkout/ShopperInput.java [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+import java.io.IOException;
+
+/**
+ * ShopperInput
+ */
+@javax.annotation.Generated(value = "io.swagger.codegen.languages.java.JavaClientCodegen", date = "2018-11-08T13:12:01.900Z[GMT]")
```
Comment (rkewlani) [inline src/main/java/com/adyen/model/checkout/ShopperInput.java]: Same as above, please remove swagger annotation.

File: src/main/java/com/adyen/service/resource/payment/Authorise3DS2.java (hunk 2 above)
Comment (AlexandrosMor) [reply src/main/java/com/adyen/service/resource/payment/Authorise3DS2.java]: Based the documentation it is used 
https://docs.adyen.com/api-explorer/#/Payment/v40/authorise3ds2

**Pull request merged**